guides/common/modules/ref_registration-methods.adoc

```diff
@@ -0,0 +1,26 @@
+[id="registration-methods_{context}"]
+= Registration Methods
+
+There are the following main methods for registering hosts:
+
+Registration templates::
++
+You generate a `curl` command from {Project} and run this command from an unlimited number of hosts to register them using the global registration template.
```

Comment:
We have two templates that we use in the registration process, I would say
`to register them using the global registration`